examples/self-pod-monitoring.yaml

```diff
@@ -26,9 +26,14 @@ spec:
       app.kubernetes.io/name: collector
   endpoints:
   - port: prom-metrics
-    interval: 10s
```

Comment:
Is this scrape interval reduction worth it? People are complaining about the cardinality of less-useful metrics, not the scrape interval.
I'm not sure why these were set to 10s originally - perhaps it's more important to have this fast of a feedback cycle when it comes to self-evaluation - but we should figure out the Chesterton's Fence here.

Reply:
Good point. I see the [Prometheus example uses 15s.](https://github.com/prometheus/prometheus/blob/501bc6419ee54ee5617d75c3bc16ecf36d94c552/documentation/examples/prometheus.yml#L3-L4)

@bwplotka what do you think?

Reply:
Yea. My understanding was that this is a "useful example for debugging GMP cases, by GMP (e.g. prometheus-engine) advanced users, devs/contributors." - that's why shorter interval, which is not needed for long term scraping.

However, we see users using this thinking it's mandatory or "generally" useful. No one blocks them, but they would be fine with 30s or even 60s.

Reply:
OK, then let's do 30s